### PR TITLE
Grant contributions

### PR DESCRIPTION
**Pull request by rubelux**

### CODE REVIEW

File: components/Grant/Support/TiersBox.js [hunk 1]
```diff
@@ -22,18 +22,21 @@ const eventBoxStyles = makeStyles(() => ({
 }))
 
 const tiersToShow = (tiers, currentSubscription) => {
-    console.log('currentSubscription', !currentSubscription.tier)
-    if (!currentSubscription || !currentSubscription.tier) {
-        console.log('it gets here')
+    console.log('currentSubscription', currentSubscription)
```
Comment (nobletom123) [inline components/Grant/Support/TiersBox.js:25]: Console log needed?

File: components/Grant/Support/BankdetailsForm.js [hunk 2]
```diff
@@ -38,22 +67,30 @@ const StripeLogo = () => (
...
+    inputFieldsState = {
+        fullName: false,
+        ccnumber: false,
+        expiration: false,
+        zip: false,
+    }
+
     constructor(props) {
         super(props)
+        console.log('BankdetailsForm   ')
```
Comment (nobletom123) [inline components/Grant/Support/BankdetailsForm.js:79]: Console log needed?

File: pages/api/create-subscription.js [hunk 3]
```diff
@@ -44,23 +46,45 @@ export default auth0.requireAuthentication(async function createSubscription(
...
             userId,
             userEmail,
             customerFullName,
-            coupon,
         })
 
-        subscriptionR = subscription
         customerIdR = customerId
+
+        console.log('customerIdR', customerIdR)
```
Comment (nobletom123) [inline pages/api/create-subscription.js:60]: Console log needed?

File: pages/api/get-stripe-user.js [hunk 4]
```diff
@@ -10,6 +10,8 @@ const stripe = require('stripe')(stripeApiN)
 export default auth0.requireAuthentication(async function newUser(req, res) {
     const { stripeCustomerId } = req.body
 
+    console.log('stripeCustomerId', stripeCustomerId)
```
Comment (nobletom123) [inline pages/api/get-stripe-user.js:13]: Console log needed?

File: pages/api/update-subscription.js [hunk 5]
```diff
@@ -19,24 +20,28 @@ export default auth0.requireAuthentication(async function newUser(req, res) {
...
+    const downgratingToFreeTier = !priceId && stripeSubscriptionId
+    const updatingPaidTiers = priceId && stripeSubscriptionId
+    const upgratingTierFromFreeToPaid = priceId && !stripeSubscriptionId
 
     let stripeSubscription
-    let customerId = stripeSubscriptionCustomerId
+    let customerId = stripeCustomerId
+
+    console.log('stripeCustomerId  ', stripeCustomerId)
+    console.log('upgratingTierFromFreeToPaid  ', upgratingTierFromFreeToPaid)
```
Comment (nobletom123) [inline pages/api/update-subscription.js:44]: Console log needed?

File: pages/api/update-subscription.js [hunk 6]
```diff
@@ -82,11 +88,10 @@ export default auth0.requireAuthentication(async function newUser(req, res) {
     const newSubmissionAllowance = availableSubmissions + allowedSubmissions
 
     // check customer's stripe id
-    if (stripeSubscriptionCustomerId) {
+    if (stripeCustomerId) {
+        console.log('stripeCustomerId in here    ', stripeCustomerId)
```
Comment (nobletom123) [inline pages/api/update-subscription.js:92]: Console log needed?

File: pages/api/update-subscription.js [hunk 7]
```diff
@@ -99,49 +104,79 @@ export default auth0.requireAuthentication(async function newUser(req, res) {
             subscriptions: { data },
         } = customer
 
-        const stripeSubscriptionItemSavedId = data[0].items.data[0].id
+        console.log('subscriptions data', data)
```
Comment (nobletom123) [inline pages/api/update-subscription.js:107]: Console log needed?

File: pages/api/update-subscription.js [hunk 8]
```diff
@@ -99,49 +104,79 @@ export default auth0.requireAuthentication(async function newUser(req, res) {
...
     if (upgratingTierFromFreeToPaid) {
-        const {
-            subscription,
-            customerId: newCustomerId,
-        } = await createStripeCustomerAndSubscription({
-            priceId,
+        console.log(
+            'it gets to upgratingTierFromFreeToPaid  ',
+            upgratingTierFromFreeToPaid
+        )
```
Comment (nobletom123) [inline pages/api/update-subscription.js:140]: Console log needed?

File: pages/api/update-subscription.js [hunk 9]
```diff
@@ -99,49 +104,79 @@ export default auth0.requireAuthentication(async function newUser(req, res) {
...
+        if (errorUpdatingUser) {
+            return res.status(500).json({ error: 'error updating user' })
         }
-
-        customerId = null
     }
 
     // update subscription from paid tier to another
     if (updatingPaidTiers) {
+        console.log('updatingPaidTiers  ', updatingPaidTiers)
```
Comment (nobletom123) [inline pages/api/update-subscription.js:179]: Console log needed?

File: pages/api/update-subscription.js [hunk 10]
```diff
@@ -162,6 +197,9 @@ export default auth0.requireAuthentication(async function newUser(req, res) {
         stripeSubscription = await updateSubscription
     }
 
+    console.log('updating tier', newTierId)
+    console.log('update stripeSubscription   ', stripeSubscription)
```
Comment (nobletom123) [inline pages/api/update-subscription.js:201]: Console log needed?

File: pages/grant/[...param].js [hunk 11]
```diff
@@ -96,6 +96,8 @@ const Grant = (props) => {
                                 return <div />
                             }
 
+                            console.log('data', data)
```
Comment (nobletom123) [inline pages/grant/[...param].js:99]: Console log needed?

File: components/Grant/GrantBox.js [hunk 12]
```diff
@@ -203,6 +204,10 @@ const GrantBox = ({
     const [showOnlyMemberPopup, setShowOnlyMemberPopUp] = useState(false)
     const showVideoStreaming =
         externalLink && externalLink.length > 0 && fullpage
+
+    console.log('externalLink   ', externalLink)
+    console.log('fullpage   ', fullpage)
```
Comment (nobletom123) [inline components/Grant/GrantBox.js:209]: Console log needed?

File: components/Grant/GrantBox.js [hunk 13]
```diff
@@ -259,6 +265,8 @@ const GrantBox = ({
 
     const SUPPORTERS = data ? [...getUsersFroMember(members, 'SUPPORTER')] : []
 
+    console.log('showVideoStreaming    ', showVideoStreaming)
```
Comment (nobletom123) [inline components/Grant/GrantBox.js:268]: Console log needed?

**Review by nobletom123 — COMMENTED** [review]
Quite a few console logs found - did you want to keep them in the code?

**Comment by rubelux** [comment]
@nobletom123 thanks for flagging them. All deleted plus many more

**Review by nobletom123 — APPROVED** [review]

**Pull request merged**